File: Respostas/Bee_1011/1011.c
#include <stdio.h>
#include <math.h>

int main() {
	double raio;
	
	scanf("%lf", &raio);
	
	printf("VOLUME = %.3lf\n", (4.0/3) * 3.14159 * (pow(raio, 3)));
	
	return 0;
}

File: Respostas/Bee_1011/1011.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) throws IOException {
		double raio;
		
		Scanner input = new Scanner(System.in);
		
		raio = input.nextDouble();
		
		System.out.printf("VOLUME = %.3f\n", (4.0 / 3) * 3.14159 * (Math.pow(raio, 3)));
	}
}